action.yml

```diff
@@ -92,8 +92,7 @@ runs:
       run: |
         pyinstaller \
           --clean \
-          --noconfirm \
-          --dist ${{ inputs.exe_path }} \
+          --distpath ${{ inputs.exe_path }} \
```

Comment:
This change is not necessary as it is same as earlier

Reply:
I don't remember making this change, it may have been the other contributor to the Martin fork, however, the correct argument based on the PyInstaller documentation is "--distpath" and not "--dist" :)

Reply:
ok, then just let it be.